base/data/gm4-1.2/functions/log_start.mcfunction

```diff
@@ -1,4 +1,5 @@
 scoreboard players set @a[gamemode=creative] gm4_creative 0
+execute as @a[gamemode=!creative] unless score @s gm4_creative matches 0.. run scoreboard players operation @s gm4_creative = $cooldown gm4_creative
```

Comment:
Shouldn't `$cooldown` be `#cooldown` as it is a constant? Our current use case for `#` is
> fake players used to store numbers for scoreboard operations should be named as a "#" followed by that number

However, I've been using `#` to marks constants in general.
Not that it would matter a lot, most of the current Gamemode 4 modules do not currently follow these standards but we should aim to change this over time.